.github/workflows/packaging-test-pipelines.yml

```diff
@@ -0,0 +1,112 @@
+name: Packaging Tests
+
+on:
+  push:
+    branches: "**"
+
+  workflow_dispatch:
+
+jobs:
+
+  get_postgres_versions_from_file:
+    runs-on: ubuntu-latest
+    outputs:
+      pg_versions: ${{ steps.get-postgres-versions.outputs.pg_versions }}
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+        with:
+          fetch-depth: 2
+      - name: Get Postgres Versions
+        id: get-postgres-versions
+        run: |
+          # Postgres versions are stored in .circleci/config.yml file in "build-[pg-version] format. Below command
+          # extracts the versions and get the unique values.
+          pg_versions=`grep -Eo 'build-[[:digit:]]{2}' .circleci/config.yml|sed -e "s/^build-//"|sort|uniq|tr '\n' ','| head -c -1`
+          pg_versions_array="[ ${pg_versions} ]"
+          echo "Supported PG Versions: ${pg_versions_array}"
+          # Below line is needed to set the output variable to be used in the next job
+          echo "pg_versions=${pg_versions_array}" >> $GITHUB_OUTPUT
+
+  build_tests_in_packaging_images:
+    name: tests_build_in_packaging_images
+    needs: get_postgres_versions_from_file
+    runs-on: ubuntu-latest
+    strategy:
+      fail-fast: false
+      matrix:
+        # While we use separate images for different Postgres versions in rpm
+        # based distros, we use the same docker image for builds based on
+        # different Postgres versions in deb based distros.
+        # For this reason, while we need to use a "matrix" to generate names of
+        # rpm images, we can simply include deb based ones into "packaging_docker_image"
+        # list.
+        packaging_docker_image:
+          - citus/packaging:oraclelinux-7-pg
+          - citus/packaging:oraclelinux-8-pg
+          - citus/packaging:centos-7-pg
+          - citus/packaging:centos-8-pg
+          - citus/packaging:almalinux-9-pg
+        POSTGRES_VERSION: ${{ fromJson(needs.get_postgres_versions_from_file.outputs.pg_versions) }}
+        include:
+          - packaging_docker_image: citus/packaging:ubuntu-jammy-all
+          - packaging_docker_image: citus/packaging:debian-bookworm-all
+          - packaging_docker_image: citus/packaging:ubuntu-focal-all
+          - packaging_docker_image: citus/packaging:debian-bullseye-all
+          - packaging_docker_image: citus/packaging:ubuntu-bionic-all
+          - packaging_docker_image: citus/packaging:debian-buster-all
+
+    container:
+      image: ${{ matrix.packaging_docker_image }}${{ matrix.POSTGRES_VERSION }}
+
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@v2
+
+      - name: Add Postgres installation directory into PATH for rpm based distros
+        run: |
+          echo "/usr/pgsql-${{ matrix.POSTGRES_VERSION }}/bin" >> $GITHUB_PATH
+        if: ${{ matrix.POSTGRES_VERSION != '' }}
```

Comment:
there is no statements below this line, so should we drop this ?

Reply:
I don't understand what you mean to drop. Is it the whole step?

Reply:
I meant, what does that `if: `s statement control since I don't see any lines of code below ?

Reply:
anyway, this seems like an outdated comment